sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/Generated/QuestionAnsweringProjectsClient.cs

```diff
@@ -40,7 +40,7 @@ protected QuestionAnsweringProjectsClient()
         /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
         /// <param name="options"> The options for configuring the client. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> or <paramref name="credential"/> is null. </exception>
-        public QuestionAnsweringProjectsClient(Uri endpoint, AzureKeyCredential credential, QuestionAnsweringClientOptions options = null)
+        public QuestionAnsweringProjectsClient(Uri endpoint, AzureKeyCredential credential, QuestionAnsweringProjectsClientOptions options = null)
```

Comment:
We intentionally wanted to use the same ClientOptions class as the inference client. Do we need to add our own explicit constructor, or how else would that work?

/cc @AhmedLeithy @ZulaMostafa

Reply:
This PR is a draft. Thanks for pointing to the issue, fixed.

> Do we need to add our own explicit constructor, or how else would that work?

`CodeGenTypeAttribute` has effect before generation, `CodeGenSuppressTypeAttribute` - after, so the fix was to rename type back and then suppress generated part of it:
https://github.com/Azure/azure-sdk-for-net/pull/26271/files#diff-9c5c0a8d636fa122f0df610689e68b01173cf5433c669ffd863096f057b8300aR13